src/app/drivingLicence/fields.js

```diff
@@ -292,5 +292,36 @@ module.exports = {
       { value: "noLicence" }
     ],
     validate: ["required"]
+  },
+  confirmDetails: {
+    type: "radios",
+    label: "",
+    legend: "",
+    items: [{ value: "detailsConfirmed" }, { value: "detailsNotConfirmed" }],
+    validate: ["required"]
+  },
+  cydLastName: {
+    type: "text"
+  },
+  cydGivenNames: {
```

Comment:
We can remove these cyd fields now as mapping of values into the sessionModel will be handled by the work done in this PR - https://github.com/govuk-one-login/ipv-cri-dl-front/pull/306

Reply:
All cyd fields now removed from fields.js.